src/server/api/routes/batch-default-values.router.js

```diff
@@ -37,8 +37,21 @@ const {
  *        required: true
  *      - name: stage
  *        in: query
- *        type: string
- *        required: true
+ *        description: Stage default values for a batch.
+ *        required: false
+ *        type: array
+ *        items:
+ *          type: string
+ *          enum:
+ *          - current
+ *          - storage
+ *          - seeding
+ *          - propagation
+ *          - maturity
+ *          - harvest
+ *          - delivery
+ *          default: current
```

Comment:
I think by default this should be empty, not current stage

Reply:
I removed the default altogether. It has no effect on the outcome. I will leave it out.